### PR TITLE
7309 fix sorting arrows

### PR DESCRIPTION
**Pull request by TanyaGordiychuk**

### CODE REVIEW

File: conference/react-js/src/components/Talks/TalksTable/TalksTable.jsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const renderTalksHeader = columns => (
       return (<TableHeaderColumn
         key={col}
         data-name={`${col}`}
-        className="table-header__item"
+        className="table-header__item "
```
Comment (ScreamCat1) [inline conference/react-js/src/components/Talks/TalksTable/TalksTable.jsx]: Excess space. Delete it please.

File: conference/react-js/src/components/Talks/TalksTable/TalksTable.jsx [hunk 2]
```diff
@@ -106,6 +106,9 @@ const TalksTable = ({ listOfTalks, columns, onClick, sortTalks }) => (
       </TableHeader>
       <TableBody
         showRowHover
+        deselectOnClickaway={false}
+        preScanRows={false}
+
```
Comment (ScreamCat1) [inline conference/react-js/src/components/Talks/TalksTable/TalksTable.jsx]: Delete empty line please

File: conference/react-js/src/containers/Talks/talks.scss [hunk 3]
```diff
@@ -54,9 +54,19 @@
 			margin-left: 5px;
 		}
 
-		&:focus::after {
-			content: '\2BC6';
-			margin-left: 5px;
+		&__active {
```
Comment (ScreamCat1) [inline conference/react-js/src/containers/Talks/talks.scss]: Large indents.Fix it please

Comment (TanyaGordiychuk) [reply conference/react-js/src/containers/Talks/talks.scss]: There are no such fields any more

**Review by ScreamCat1 — APPROVED** [review]

File: conference/react-js/src/containers/Talks/talks.scss [hunk 4]
```diff
@@ -1,62 +1,72 @@
...
-	&__item {
-		cursor: pointer;
+  &__item {
+    cursor: pointer;
+    &::after {
+      content: '\2BC5';
+      margin-left: 5px;
+    }
+
+    &__active {
```
Comment (YaroslavPerepelychnyi) [inline conference/react-js/src/containers/Talks/talks.scss]: according to bem for modifier should use "--" or "_"

**Review by YaroslavPerepelychnyi — CHANGES_REQUESTED** [review]

File: conference/react-js/src/containers/Talks/index.jsx [hunk 5]
```diff
@@ -129,8 +129,20 @@ class Talks extends Component {
     this.doFilter();
   };
 
-  sortTalks = ({ target: { tagName, dataset: { name } } }) => {
+  sortTalks = ({ target }) => {
+    const { classList, tagName, dataset: { name } } = target;
+    const currentEl = classList.value;
     if (tagName === 'TH' && name) {
+      if (currentEl.indexOf('table-header__item_active') !== -1) {
```
Comment (SupraVlad) [inline conference/react-js/src/containers/Talks/index.jsx:136]: there is a method `includes`. it is used to define whether string/array contains defined symbol/element.
you may use it

**Review by SupraVlad — APPROVED** [review]

**Review by YaroslavPerepelychnyi — APPROVED** [review]

**Pull request merged**